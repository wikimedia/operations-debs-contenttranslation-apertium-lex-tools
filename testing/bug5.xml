<rules>
  <rule weight="1.0">
    <match lemma="el"/>
    <match lemma="estació" tags="n.f.*"><select lemma="season" tags="n.*"/></match>
    <match tags="adv"/>
    <or>
      <match lemma="sec"/>
      <match lemma="plujós"/>
      <match lemma="humit"/>
    </or>
  </rule>

</rules>
