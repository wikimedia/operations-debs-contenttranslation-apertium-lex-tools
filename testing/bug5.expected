[ 0 ]^El<det><def><f><sg>/The<det><def><f><sg>$[ \$ 1 ]^estació<n><f><sg>/season<n><sg>$[ 2 ]^més<adv>/more<adv>$[ 3 ]^plujós<adj><f><sg>/rainy<adj><sint><f><sg>$[ 4 ]^de<pr>/of<pr>/from<pr>$[  5  ]^el<det><def><m><sg>/the<det><def><m><sg>$[ 6]^any<n><m><sg>/year<n><sg>$[7 ]^ser<vbser><pri><p3><sg>/be<vbser><pri><p3><sg>$[<b>8<\/b>]^el<det><def><m><sg>/the<det><def><m><sg>$[ 9 ]^estiu<n><m><sg>/summer<n><sg>$^.<sent>/.<sent>$^.<sent>/.<sent>$[][
0 ]^EL<det><def><f><sg>/The<det><def><f><sg>$[ \$ 1 ]^estació<n><f><sg>/season<n><sg>$[ 2 ]^més<adv>/more<adv>$[ 3 ]^plujós<adj><f><sg>/rainy<adj><sint><f><sg>$[ 4 ]^de<pr>/of<pr>/from<pr>$[  5  ]^el<det><def><m><sg>/the<det><def><m><sg>$[ 6]^any<n><m><sg>/year<n><sg>$[7 ]^ser<vbser><pri><p3><sg>/be<vbser><pri><p3><sg>$[<b>8<\/b>]^el<det><def><m><sg>/the<det><def><m><sg>$[ 9 ]^estiu<n><m><sg>/summer<n><sg>$^.<sent>/.<sent>$^.<sent>/.<sent>$[][
0 ]^el<det><def><f><sg>/The<det><def><f><sg>$[ \$ 1 ]^estació<n><f><sg>/season<n><sg>$[ 2 ]^més<adv>/more<adv>$[ 3 ]^plujós<adj><f><sg>/rainy<adj><sint><f><sg>$[ 4 ]^de<pr>/of<pr>/from<pr>$[  5  ]^el<det><def><m><sg>/the<det><def><m><sg>$[ 6]^any<n><m><sg>/year<n><sg>$[7 ]^ser<vbser><pri><p3><sg>/be<vbser><pri><p3><sg>$[<b>8<\/b>]^el<det><def><m><sg>/the<det><def><m><sg>$[ 9 ]^estiu<n><m><sg>/summer<n><sg>$^.<sent>/.<sent>$^.<sent>/.<sent>$[][
]
